specification/ServiceBusManagement.json

```diff
@@ -0,0 +1,413 @@
+                  "type": "boolean"
+                },
+                "EntityAvailabilityStatus": {
+                  "description": "TODO: Add description",
+                  "type": "string"
+                },
+                "EnableExpress": {
+                  "description": "TODO: Add description",
+                  "type": "boolean"
+                }
+              },
+              "xml": {
+                "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+              }
+            }
+          }
+        }
+      },
+      "required": [
+        "id",
+        "published",
+        "created",
+        "author",
+        "link",
+        "content"
+      ],
+      "xml": {
+        "name": "entry",
+        "namespace": "http://www.w3.org/2005/Atom"
+      }
+    },
+    "CreateQueueBody": {
+      "description": "The response from a CreateQueue operation.",
+      "type": "object",
+      "properties": {
+        "content": {
+          "description": "TODO: Add description",
+          "type": "object",
+          "properties": {
+            "type": {
+              "description": "TODO: Add description",
+              "type": "string",
+              "xml": {
+                "attribute": true
+              }
+            },
+            "QueueDescription": {
+              "description": "TODO: Add description",
+              "type": "object",
+              "properties": {
+                "LockDuration": {
+                  "description": "TODO: Add description",
+                  "type": "string",
+                  "default": "PT1M"
+                },
+                "MaxSizeInMegabytes": {
+                  "description": "TODO: Add description",
+                  "type": "number",
+                  "default": 1024
+                },
+                "RequiresDuplicateDetection": {
+                  "description": "TODO: Add description",
+                  "type": "boolean",
+                  "default": false
+                },
+                "RequiresSession": {
+                  "description": "TODO: Add description",
+                  "type": "boolean",
+                  "default": false
+                },
+                "DeadLetteringOnMessageExpiration": {
+                  "description": "TODO: Add description",
+                  "type": "boolean",
+                  "default": false
+                },
+                "MaxDeliveryCount": {
+                  "description": "TODO: Add description",
+                  "type": "integer",
+                  "default": 10
+                },
+                "EnableBatchedOperations": {
+                  "description": "TODO: Add description",
+                  "type": "boolean",
+                  "default": true
+                },
+                "Status": {
+                  "description": "TODO: Add description",
+                  "type": "string",
+                  "default": "Active"
```

Comment:
Wouldn't this be readonly? If not "Active" then what are the other possible values that one can provide? Can someone create an "InActive" queue?